code/datums/components/crafting/recipes.dm

```diff
@@ -1030,6 +1030,17 @@
 	result = /obj/item/pickaxe/improvised
 	category = CAT_MISC
 
+// Skryat addition start
```

Comment:
Perhaps move this to a modular folder, no need for it to be here

Reply:
will look into this